src/openrct2/ride/coaster/AirPoweredVerticalCoaster.cpp

```diff
@@ -647,7 +653,8 @@ static void AirPoweredVerticalRCTrackVerticalSlopeUp(
             PaintAddImageAsChildRotated(
                 session, direction, trackImageId, { 0, 0, height }, { { 0, 6, height }, { 20, 32, bbHeight } });
 
-            WoodenASupportsPaintSetup(session, 0, 0, height, session.SupportColours);
```

Comment:
Is this right? Unlike the other calls, the second argument of this one was not using `direction & 1`

Reply:
Yes, this is correct - I replaced the `& 1` versions with `...Rotated(` and this occurrence with the regular version.